packages/calcite-components/src/components/shell/shell.stories.ts

```diff
@@ -360,7 +360,7 @@ darkModeRTL_TestOnly.parameters = { themes: modesDarkDefault };
 
 export const closedPanels = (): string =>
   html(`<calcite-shell content-behind>
-  <calcite-shell-panel slot="panel-start" display-mode="float">
+  <calcite-shell-panel slot="panel-start" display-mode="float-content">
```

Comment:
question: should these values remain `float` to keep test coverage for the deprecated `float` value? Applies to other changes below that are being changed from float to float-content.